src/types/voice.ts

```diff
@@ -50,8 +50,6 @@ export interface VoiceRegion {
   id: string
   /** Name of the region */
   name: string
-  /** True if this is a vip-only server */
-  vip: boolean
```

Comment:
actually set it as `false` instead of deleting it so it won't break the api

Reply:
Also handy to add @deprecate to it 😉

Reply:
I might have deleted my fork and now have to remake the pull request